doc/lua_api.md

```diff
@@ -1471,6 +1471,11 @@ Look for examples in `games/devtest` or `games/minetest_game`.
 * `allfaces`
     * Often used for partially-transparent nodes.
     * External and internal sides of textures are visible.
+    * For This drawtype textures are as follows in the exact order:
+    ```
+        tiles = {tile definition 1, def2, def3, def4, def5, def6},
+        Textures of node;       -Y,  +Z,   +X,   -Z,   +Y,   -X
```

Comment:
is this different from normal nodes?
if yes: why?
if no: we shouldn't duplicate it here